boards: beagle: beagleplay_cc1352p7: Disable 2.4g ieee802154

Ayush1325 opened the pull request:
- A lot of zephyr networking samples (zperf, http_server, etc) will complain if both 2.4g and subg ieee802154 are enabled simultaneously.
- This leads to a lot of new people getting confused with the network code not working without any clear errors.
- As we already set the alias `zephyr,ieee802154 = &ieee802154g`, I think it would be best to enable only subg by default.
- Following the example of: https://github.com/zephyrproject-rtos/zephyr/pull/78667

Ayush1325 committed:
boards: beagle: beagleplay_cc1352p7: Disable 2.4g ieee802154

- A lot of zephyr networking samples (zperf, http_server, etc) will
  complain if both 2.4g and subg ieee802154 are enabled simultaneously.
- This leads to a lot of new people getting confused with the network code
  not working without any clear errors.
- As we already set the alias `zephyr,ieee802154 = &ieee802154g`, I think
  it would be best to enable only subg by default.

Signed-off-by: Ayush Singh <ayush@beagleboard.org>

vaishnavachath commented on boards/beagle/beagleplay/beagleplay_cc1352p7.dts:
```diff
@@ -87,7 +87,7 @@
 &ieee802154 {
-	status = "okay";
+	status = "disable";
 };
```
`status = "disabled";`

Ayush1325 committed:
boards: beagle: beagleplay_cc1352p7: Fix twister

- Add UART console config.
- Fixes tests/subsys/llext/simple/llext.simple.readonly_mpu

Signed-off-by: Ayush Singh <ayush@beagleboard.org>
